packages/shared/src/utils/login/login.js

```diff
@@ -21,9 +20,11 @@ export const redirectToSignUp = ({ is_deriv_crypto }) => {
 
 export const loginUrl = ({ language }) => {
     const server_url = LocalStore.get('config.server_url');
-    const signup_device = LocalStore.get('signup_device') || (isMobileOs() ? 'mobile' : 'desktop');
```

Comment:
We don't need to set default values anymore? Does `CookieStorage` always have `signup_device`?

Reply:
@akmal-binary yes, since we are loading cookies through cookies script on load. So it will always be there

Reply:
Ok, thank you for the explanation 👍🏻